### ISSUE TITLE
Trait 'Launcher\Mercurius\MercuriusUser' not found

### ISSUE THREAD

**Issue by bilalmubarik** [launcher-host/mercurius#40]
Trait 'Launcher\Mercurius\MercuriusUser' not found" error while starting server.

## Server environment
 - PHP:      v7.1.3
 - Laravel:  v5.7
 - Database: [MySQL] [5.7.23] (e.g. MySQL 8.0)

## Description
I have followed the entire installation process given on the project page but still when I start the server. this error "Trait 'Launcher\Mercurius\MercuriusUser' not found" comes up. Seems like my new laravel installation isn't picking installing files of this project at all.

**Comment by akazorg** [comment]
Hi @bilalmubarik, try running `composer dump-autoload`. If it doesn't work, try reinstall Mercurius.

**Comment by CristianEnache** [comment]
@bilalmubarik you are correct. When installing via composer, there are a few missing files compared to the git repo. MercuriusUser.php and Message.php from what I noticed. I cloned the repo and copied the files in /vendor/launcher/mercurius/src

**Comment by akazorg** [comment]
Thanks for confirming this issue @CristianEnache

**Comment by impactcolor** [comment]
@CristianEnache great catch, this worked for me

**Comment by abrhambas01** [comment]
after doing such ?
